lib/websocket-server.js

```diff
@@ -275,6 +275,16 @@ class WebSocketServer extends EventEmitter {
 
     cb(ws);
   }
+
+  /**
+   * Provide address of server
```

Comment:
We need a better description and specify what is returned via `@return ...`. Copying https://nodejs.org/dist/latest/docs/api/net.html#net_server_address is ok.

Reply:
done sir :)

Reply:
Noooo CI fail

Reply:
Please put the description on this line and wrap lines at 80 chars.

```
/**
 * Returns the bound address, the address family name, and port of the server
 * as reported by the operating system if listening on an IP socket.
 * If the server is listening on a pipe or UNIX domain socket, the name is
 * returned as a string.
 *
 * @return {(Object|String|null)} The address of the server
 * @public
 */
```